com.github.hopsan.Hopsan.json.in

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "com.github.hopsan.Hopsan",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.12",
+    "sdk": "org.kde.Sdk",
+    "command": "hopsangui",
+    "modules": [
+        {
+            "name": "doxygen",
+            "buildsystem": "cmake",
+            "config-opts": [ "-DCMAKE_BUILD_TYPE=Release" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/doxygen/doxygen/archive/Release_1_8_15.tar.gz",
+                    "sha256": "cc5492b3e2d1801ae823c88e0e7a38caee61a42303587e987142fe9b68a43078"
+                }
+            ],
+            "cleanup": [ "*" ]
+        },
+        {
+            "name": "hopsan",
+            "disabled": false,
+            "buildsystem": "simple",
+            "build-commands": [
+                /* Configure source code with version numbers and build settings */
+                "./buildDebPackage/prepareSourceCode.sh $(pwd) $(pwd) HOPSAN_BASE_VERSION HOPSAN_RELEASE_REVISION HOPSAN_FULL_RELEASE_VERSION HOPSAN_DEVELOPMENT_RELEASE false",
+                /* Since Hopsan already has built-in dependency module build scripts we reuse them instead of duplicating the work with flatpak modules */
+                "cd Dependencies; ./setupDiscount.sh",
```

Comment:
Is there a reason for having those two files here?

Reply:
I assume you are referring to the Hopsan.json.in and makeFlatpakRelease script. I want to automate as much as possible when generating a new release, some modifications to the build script is needed, to "prepare" the source code. Mostly related to version information and stable/development build. I have similar solutions for .deb, snap and Windows packaging.
Right now it is easiest for me to have the files here, but I will consider moving them upstream as a future improvement.